src/content/docs/synthetics/synthetic-monitoring/using-monitors/monitor-downtimes-disable-monitoring-during-scheduled-maintenance-times.mdx

```diff
@@ -35,7 +41,7 @@ Create recurring monitor downtimes for routine monitor maintenance or regularly
 2. Select **<Icon name="fe-plus-circle"/>
    Create monitor downtime**.
 3. Specify a name for the monitor downtime.
-4. Select a daily, weekly, or monthly frequency. Choose additional scheduling options, such as day of the week, time of day, and how long the window will last.
+4. Select a daily, weekly, or monthly frequency. Choose additional scheduling options, such as day of the week, time of day, and how long the window will last. 
```

Comment:
I'd tweak slightly:

"Select a daily, weekly, or monthly frequency. Choose additional scheduling options, such as day of the week, time of day, and how long the window lasts."

Reply:
Sounds good. I pushed a commit with this change.